projects/core/src/error-handling/cx-error-handler.ts

```diff
@@ -27,33 +26,26 @@ import {
  */
 @Injectable()
 export class CxErrorHandler implements ErrorHandler {
-  //TODO: Verify whether feature flag in this case is justified.
-  // if yes - Add Jira ticket to remove in next major release or after 6 months
-  private featureConfig = inject(FeatureConfigService);
-  logger = inject(LoggerService);
+  /**
+   * @deprecated Since 6.6
+   */
+  protected logger = inject(LoggerService);
+  protected injector = inject(Injector);
 
-  injector = inject(Injector);
-
-  errorInterceptors: ErrorInterceptor[] = sortErrorInterceptors(
+  protected errorInterceptors: ErrorInterceptor[] = sortErrorInterceptors(
     this.injector.get(ERROR_INTERCEPTORS, [])
   );
 
   handleError(error: Error): void {
-    //TODO: Verify whether feature flag in this case is justified.
-    // if yes - Add Jira ticket to remove in next major release or after 6 months
-    if (this.featureConfig.isLevel('6.6')) {
-      // Similar to Angular's interceptors, error interceptors are organized from right to left,
-      // ensuring that the ultimate execution order is from left to right.
-      // In other words, if the interceptors array contains `[a, b, c]`,
-      // our goal is to create a chain that can be envisioned as c(b(a(end))),
-      // constructed by progressively adding elements from the innermost to the outermost.
-      const chain = this.errorInterceptors.reduceRight(
-        (next, interceptor) => handleInterceptors(next, interceptor),
-        tailChain
-      );
-      chain(error);
-    } else {
-      this.logger.error(error);
-    }
+    // Similar to Angular's interceptors, error interceptors are organized from right to left,
+    // ensuring that the ultimate execution order is from left to right.
+    // In other words, if the interceptors array contains `[a, b, c]`,
+    // our goal is to create a chain that can be envisioned as c(b(a(end))),
+    // constructed by progressively adding elements from the innermost to the outermost.
+    const chain = this.errorInterceptors.reduceRight(
+      (next, interceptor) => handleInterceptors(next, interceptor),
+      tailChain
```

Comment:
I believe it should be simply noop
```suggestion
      ()=>{}
```

Reply:
Even if the function does nothing, I like its name and description is related to the error interceptor chain. I'd like to keep it as it is.